src/database/FirehoseConnector.js

```diff
@@ -152,6 +154,22 @@ class FirehoseConnector {
 
         this._putRecord(schemaObj, this._meetingStatsStream);
 
+        Object.keys(e2epings).forEach(ping => {
```

Comment:
neat pick: maybe change the variable name to remoteEndpointId or endpointId?

Reply:
Yes that actually makes a lot of sense. endpointId was already take, so remoteEndpointId it is.